### DISCUSSION TITLE
Multiple Customer environments / credentials config-repository

### DISCUSSION THREAD

**Discussion by stefanfranken** [microsoft/Dual-write-automations#44] [Ideas]
Hi,
I´m using DW-automation (excellent tool!) for managing several customers, with different logins and configuration settings. Currently I have to handle each customer with my own DW-automation application folder (duplicated everything). In case of a Release Update, I have to maintain all folders.
Idea:
- Make Customer with credentials and config-file repository directory configurable

**Comment by JoPe72** [comment]
I agree... this would be great. Also being able to backup the environment list to be able to move it would be great
